client/src/Q&As/SearchQandA.jsx

```diff
@@ -22,38 +21,58 @@ const Questions = ({ product, setDateFormat }) => {
       });
   }, [product.id]);
 
+  const sortQuestions = () => {
+    const sorted = questions.sort((a, b) => {
+      return b.question_helpfulness > a.question_helpfulness;
+    });
+    setQuestions(sorted);
+  };
 
-  const loadMore = useCallback(() => {
-    setLoadPage(prev => prev + 2);
-  }, []);
-
-  const loadQuestions = questions.slice(0, loadPage).map((question) => {
+  const filteredQuestion = questions.filter((val) => {
+    if (search === '' || search.length < 3 ) {
+      return val;
+    } else if (val.question_body.toLowerCase().includes(search.toLowerCase())) {
+      return val;
+    }
+  }).slice(0, loadPage).map((question, key) => {
     return (
       <div className="questions_div" key={question.question_id}>
         <Styles.QuestionsContainer>
-          Q: {question.question_body} <Styles.addAnswerButton onClick={() => setShow(true)} >Add Answer</Styles.addAnswerButton>
-          <Modal title="Submit Your Answer" subTitle={product.name} id={question.question_id} questionBody={question.question_body} show={show} onClose={() => setShow(false)}/><Helpful origin="qa/questions" id={question.question_id} helpCount={question.question_helpfulness}/><br/>
+          Q: {question.question_body} <Styles.addAnswerButton onClick={() => setShow(true)} >Add Answer</Styles.addAnswerButton><Helpful origin="qa/questions" id={question.question_id} helpCount={question.question_helpfulness}/>
+          <Modal title="Submit Your Answer" subTitle={product.name} id={question.question_id} questionBody={question.question_body} show={show} onClose={() => setShow(false)}/><br/>
```

Comment:
Consider adding new lines to make 41 and 42 more readable

Reply:
fixed it! :)